homeassistant/components/poolstation/const.py

```diff
@@ -0,0 +1,7 @@
+"""Constants for the Poolstation integration."""
+DOMAIN = "poolstation"
+
+TOKEN = "token"
+
+COORDINATORS = "coordinators"
```

Comment:
These are rather generic, if they are not available under homeassistant.const, maybe the should be introduced there?

Reply:
There's a `token` in `CONF_TOKEN` defined homeassistant.const, I could use that one, `but coordinators` is not present. I don't have an opinion, as I mentioned it's my first pull request and my first time writing pyton. I'm happy to add it if y'all deem it's the best thing to do. I tried to not makes changes outside my integration because in short, I have no idea what I'm doing.

Reply:
:+1: it's better not to make changes outside the integration indeed.